### PR TITLE
Add slug for product table and pass into the url

### PR DESCRIPTION
**Pull request by kams2391**
add cart detail page

### CODE REVIEW

File: resources/views/components/cart-nav.blade.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
+@if (Route::has('login'))
+    @auth
+         <section class="bg-white border-b border-solid border-[#E0E0E0]">
+    <div class="container mx-auto">
+      <div class="max-w-screen-xl mx-auto">
+        <div class="flex items-center justify-between">
+          <div class="text-base font-medium text-center text-[#000]">
+            <ul class="flex flex-wrap -mb-px">
+              <li class="me-2">
+                <a href="{{ url('/cart') }}"
```
Comment (upworksanjeev) [inline resources/views/components/cart-nav.blade.php:10]: @kams2391 Assign name to the route and use it here
i.e {{ route('cart') }}

Comment (upworksanjeev) [reply resources/views/components/cart-nav.blade.php:10]: Do this for all link

File: app/Http/Controllers/CartController.php [hunk 2]
```diff
@@ -48,15 +49,57 @@ public function index(Request $request)
...
+	} 
+	
+	 /**
+     * Quick Order Entry details
+     */
+	public function quickEntry(Request $request)
+    {
+		$user = Auth::user();
+		
+		if(isset($request->product_id)){
```
[the remark this answers is not in the document]
Comment (kams2391) [reply app/Http/Controllers/CartController.php]: Done

File: app/Http/Controllers/CartController.php [hunk 3]
```diff
@@ -48,15 +49,57 @@ public function index(Request $request)
...
+		$cart=Cart::where('user_id', $user->id)->first();
+		if(isset($cart->id)){
```
[the remark this answers is not in the document]
Comment (kams2391) [reply app/Http/Controllers/CartController.php]: Done

File: app/Http/Controllers/CartController.php [hunk 4]
```diff
@@ -48,15 +49,57 @@ public function index(Request $request)
...
+			Cart::where('id', $cart->id)->update(['total_items' => $cart->total_items+1]);
```
[the remark this answers is not in the document]
Comment (kams2391) [reply app/Http/Controllers/CartController.php]: Done and changed in other code also.

File: routes/web.php [hunk 5]
```diff
@@ -37,6 +37,8 @@
     Route::delete('/profile', [ProfileController::class, 'destroy'])->name('profile.destroy');
 	Route::post('/cart', [CartController::class, 'index']);
 	Route::get('/cart', [CartController::class, 'index']);
+	Route::post('/quickentry', [CartController::class, 'quickEntry']);
```
[the remark this answers is not in the document]
Comment (kams2391) [reply routes/web.php]: Done

File: resources/views/quick-entry.blade.php
[the remark this answers is not in the document]
Comment (kams2391) [reply resources/views/quick-entry.blade.php:1]: Made cart list dynamic and others are static

File: resources/views/components/cart-nav.blade.php [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+                    <svg class="w-4 h-4" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none"
+                      viewBox="0 0 20 20">
+                      <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                        d="m19 19-4-4m0-7A7 7 0 1 1 1 8a7 7 0 0 1 14 0Z" />
+                    </svg>
+                    <span class="sr-only">Search</span>
+                  </button>
+                  <button id="dropdown-button" data-dropdown-toggle="dropdown"
+                    class="flex-shrink-0 z-10 inline-flex items-center py-2.5 pe-4 text-sm font-medium text-center text-white bg-[#494949] rounded-e-lg"
+                    type="button"><svg class="w-2.5 h-2.5 ms-2.5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg"
```
[the remark this answers is not in the document]
Comment (kams2391) [reply resources/views/components/cart-nav.blade.php]: Done created svg component file for all svg througout the site.

File: resources/views/components/cart-nav.blade.php (hunk 1 above)
Comment (kams2391) [reply resources/views/components/cart-nav.blade.php]: Done. Added name for all links.

File: resources/views/components/svg.blade.php
Comment (upworksanjeev) [inline resources/views/components/svg.blade.php:1]: @kams2391 you have added all svg icon in one file. Please create a seprate file for all icons
You can check example here: https://prnt.sc/v2Z2nU4QcHjh

**Review by upworksanjeev — APPROVED** [review]

File: resources/views/components/svg.blade.php
Comment (kams2391) [reply resources/views/components/svg.blade.php:1]: done

**Pull request merged**